coralproject/talk
# IE & Edge fixes

## What does this PR do?
- Fix IE11 Popup issues
- Fix IE11 Caching issues
- Fix broken spinner by using a degraded spinner for IE & Edge browsers
- Add IE and Edge browser to browserstack testing

## How do I test this PR?

- Check Login + Logout works in IE11 flawlessly
- Check degraded Spinner appear in IE11 and Edge
